src/test/java/net/datafaker/FakerTest.java

```diff
@@ -310,6 +312,18 @@ void differentLocalesTest() {
         }
     }
 
+    @Test
+    @Timeout(value = 100_000, unit = TimeUnit.MILLISECONDS)
```

Comment:
before the fix this test was able to reproduce OOM on my laptop
it takes about 3s on my laptop in case it is executed alone and about 10 s in multithreaded junit5 test run
100 s just for safety

Reply:
Why not actually use 100 seconds?

Reply:
seconds is default timeunit in junit5 and as a result IntellijIdea tends to remove it when seconds are specified as a unit...
At the same time no unit means seconds in junit5 and millis in junit4... 

i think we can put just 1 min here to simplify it and to not allow Intellijidea introducing confusion

Reply:
Sure that should still be lots.